suite-common/firmware/src/hooks/useFirmwareInstallation.ts

```diff
@@ -208,15 +206,25 @@ export const useFirmwareInstallation = (
 
         // Automatically restarting from bootloader to normal mode at the end of non-intermediary installation:
         if (reconnectEvent?.method === 'wait' || reconnectEvent?.method === 'auto') {
-            return { operation: 'restarting', progress: 100 };
+            return {
+                operation: 'restarting',
+                // NOTE: have progress 0 when restarting before the installation
+                progress: deviceIsWaitingForConfirmationToConnectToHost ? 0 : 100,
```

Comment:
this is wrong btw. there should be also a check , if this is a THP device.

I am thinking if there is a way to know if the installation started or not @komret idea?